ts-defs-src/test-api/structure.d.ts

```diff
@@ -110,6 +110,10 @@ interface FixtureFn {
      * Disables page reloading which would happen right before each test in this fixture.
      */
     disablePageReloads: this;
+    /**
+     * Disables global concurrency setting for this fixture.
```

Comment:
@titerman , please, review the description

Reply:
```suggestion
     * Disables the global concurrency setting for this fixture.
```